### PR TITLE
Datawrapper iframe component w/ event interaction

### PR DESCRIPTION
**Pull request by benkates**
Adds a Datawrapper Iframe component that bring in Datawrapper's [event interaction library](https://developer.datawrapper.de/docs/listening-to-chart-interaction-events) specifically for map datawrapper visualizations.

### CODE REVIEW

File: src/lib/DatawrapperIframe/DatawrapperIframe.svelte [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    if (void 0 !== a.data["datawrapper-height"]) {
+      var e = document.querySelectorAll("iframe");
+      for (var t in a.data["datawrapper-height"])
+        for (var r = 0; r < e.length; r++)
+          if (e[r].contentWindow === a.source) {
+            var i = a.data["datawrapper-height"][t] + "px";
+            e[r].style.height = i;
+          }
+    }
+  });
```
Comment (mitchthorson) [inline src/lib/DatawrapperIframe/DatawrapperIframe.svelte:48]: let's wrap this in a conditional check for window:
```js
if (typeof window !== "undefined") {}
```
so it doesn't bomb during ssr

File: src/lib/DatawrapperIframe/events.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+// https://developer.datawrapper.de/docs/listening-to-chart-interaction-events
+
+if (typeof window.datawrapper === "undefined") window.datawrapper = {};
```
Comment (mitchthorson) [inline src/lib/DatawrapperIframe/events.js:3]: wrap in `window` check

File: src/lib/DatawrapperIframe/events.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+var datawrapper = window.datawrapper;
+var listeners = {};
+
+window.addEventListener("message", receiveMessage, false);
```
Comment (mitchthorson) [inline src/lib/DatawrapperIframe/events.js:8]: wrap in `window` check

File: src/lib/DatawrapperIframe/datawrapper-event-list.json
Comment (mitchthorson) [inline src/lib/DatawrapperIframe/datawrapper-event-list.json:1]: smart!

**Review by mitchthorson — CHANGES_REQUESTED** [review]

**Comment by benkates** [comment]
@mitchthorson added window checks! Should be good to go!

**Review by mitchthorson — APPROVED** [review]
LGTM!

**Pull request merged**